src/EA.Weee.RequestHandlers/Mappings/ReturnMap.cs

```diff
@@ -35,7 +37,8 @@ public ReturnData Map(ReturnQuarterWindow source)
                 CreatedDate = source.Return.CreatedDate,
```

Comment:
Nothing you've changed, but apparently SchemeDataItems has no unit test attached to it?

Reply:
Added unit test for this property